lib/mayaUsd/CMakeLists.txt

```diff
@@ -219,10 +219,13 @@ mayaUsd_compile_config(${PROJECT_NAME})
 # include directories
 # -----------------------------------------------------------------------------
 target_include_directories(${PROJECT_NAME} 
-    PUBLIC
-        ${MAYA_INCLUDE_DIRS}
+    SYSTEM PUBLIC
         ${PXR_INCLUDE_DIRS}
+        ${MAYA_INCLUDE_DIRS}
         ${UFE_INCLUDE_DIR}
+)
+target_include_directories(${PROJECT_NAME} 
+    PUBLIC
         ${CMAKE_BINARY_DIR}/include
 )
```

Comment:
USD include paths were already set as system (by pxrConfig.cmake). According to cmake docs the system paths **always** come after regular paths. So need to set Maya as a system path as well and put if after USD so that USD will come first. This will make our build find the TBB from the USD artifact rather than Maya which will be updating to OneTBB.